src/components/header/header.css

```diff
@@ -0,0 +1,174 @@
+header {
+  box-sizing: border-box;
+  height: var(--size-10);
+  width: auto;
+  border: var(--border-size-1) dotted #989898ca;
```

Comment:
@Auhseh 
Just curious, I see a few of these 8 character hex values, is this a new extension in CSS?  Linters and VSCode don't seem to complain but I'm probably just a little out of touch, hah.

Aside from that, it does look like there are a couple of these repeated values through this CSS, perhaps we could create some locally scoped values using `:host` selector at the top of this file?
```css
:host {
  --border-color-gray: #989898ca
}
```

Or they can be added to _theme.css_ if we expect to use them in other places too.

Reply:
Sorry for the lag here, I was unwell and I’m just now recuperating… I’m back now.

The extra two characters essentially just tweak the opacity..
The :host idea looks neat